### PR TITLE
Some small style fixes

### PR DESCRIPTION
**Pull request by chick**
simple changes found by intellij

### CODE REVIEW

File: src/test/scala/firrtlTests/DriverSpec.scala [hunk 1]
```diff
@@ -57,7 +57,7 @@ class DriverSpec extends FreeSpec with Matchers with BackendCompilationUtilities
     "options include by default a list of strings that are returned in commonOptions.programArgs" in {
       val optionsManager = new ExecutionOptionsManager("test")
 
-      optionsManager.parse(Array("--top-name", "dog", "fox", "tardigrade", "stomatopod")) should be (true)
+      optionsManager.parse(Array("--top-name", "dog", "fox", "0tardigrade", "stomatopod")) should be (true)
```
Comment (jackkoenig) [inline src/test/scala/firrtlTests/DriverSpec.scala]: why this change? These look IntelliJ suggested but this one is strange

**Comment by chick** [comment]
dang, dumb typo.  fixing
